docs/current/guides/757394-use-services.md

```diff
@@ -0,0 +1,326 @@
+---
+slug: /757394/use-services
+displayed_sidebar: "current"
+category: "guides"
+tags: ["go", "python", "nodejs"]
+authors: ["Alex Suraci"]
+date: "2023-03-07"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Embed from '@site/src/components/atoms/embed.js'
+
+# Use Services in Dagger
+
+## Introduction
+
+Dagger v0.4 introduced services, aka container-to-container networking. This feature enables users to spin up additional long-running services (as containers) and communicate with those services from their Dagger pipelines.
```

Comment:
I'd suggest to change to "[Dagger v.0.4.0](https://github.com/dagger/dagger/releases/tag/v0.4.0)" with link to the release notes.

Reply:
Done